src/main/java/com/princexml/wrapper/Prince.java

```diff
@@ -353,49 +497,87 @@ public void setCssDpi(int cssDpi) {
     //endregion
 
     //region Raster output options.
+    /**
+     * Specify the format for the raster output.
+     * @param rasterFormat The format for the raster output.
+     */
     public void setRasterFormat(RasterFormat rasterFormat) {
         this.rasterFormat = rasterFormat;
     }
 
+    /**
+     * Specify the level of JPEG compression when generating raster output in JPEG format.
```

Comment:
Should it mention "percentage" or "from 0-100" or something? Might help to know the default, too.

Reply:
I'll add in the default value. The method throws an exception if it's outside the range [0, 100], but I'll add it into the description as well.